internal/pkg/consensus/processor.go

```diff
@@ -567,18 +592,26 @@ func NewDefaultBlockRewarder() *DefaultBlockRewarder {
 var _ BlockRewarder = (*DefaultBlockRewarder)(nil)
 
 // BlockReward transfers the block reward from the network actor to the miner's owner.
-func (br *DefaultBlockRewarder) BlockReward(ctx context.Context, st state.Tree, minerOwnerAddr address.Address) error {
+func (br *DefaultBlockRewarder) BlockReward(ctx context.Context, st state.Tree, vms vm.StorageMap, minerOwnerAddr address.Address) error {
 	cachedTree := state.NewCachedTree(st)
```

Comment:
Not related to this PR, but tracing through how rewards are calculated (some in ApplyMessage, some in ApplyMessageAndPayRewards) strengthens my impression that these functions have grown to become significantly more complicated than they inherently need to be.